extension/src/popup/components/manageAssets/ChooseAsset/index.tsx

```diff
@@ -2,6 +2,7 @@ import React, { useEffect, useRef, useState } from "react";
 import { useSelector } from "react-redux";
 import StellarSdk from "stellar-sdk";
 import { Link } from "react-router-dom";
+import { Icon, Loader } from "@stellar/design-system";
```

Comment:
also the ChooseAsset view w/ a large amount of assets takes a second to load, so I added a loader to it while waiting